src/users/users.controller.ts

```diff
@@ -110,4 +112,34 @@ export class UsersController {
     const { user } = request;
     return this.usersService.getFollowingsFeed(user);
   }
+
+  @ApiSecurity('Authentication')
+  @UseGuards(JwtAuthenticationGuard)
+  @ApiOperation({ summary: 'Ban user' })
+  @ApiResponse({ status: 201, description: 'Return list whom logged user banned', type: [User] })
+  @Post('/ban')
+  ban(@Body() dto: BanUserDto, @Req() request: RequestWithUser): Promise<User[]> {
+    const { user } = request;
+    return this.usersService.banUser(dto, user);
+  }
+
+  @ApiSecurity('Authentication')
+  @UseGuards(JwtAuthenticationGuard)
+  @ApiOperation({ summary: 'Unban user' })
+  @ApiResponse({ status: 201, description: 'Return User Owner', type: [User] })
```

Comment:
Return value in description doesn't match the actual returned value. The same issue in unfollow method (81 line)